lib/generators/blacklight/solr5_generator.rb

```diff
@@ -16,6 +23,14 @@ def install_solrwrapper
       append_to_file "Rakefile", "\nrequire 'solr_wrapper/rake_task' unless Rails.env.production?\n"
     end
 
+    def copy_solr_conf
+      directory 'solr', 'solr'
+    end
+
+    def solr_wrapper_config
+      copy_file '.solr_wrapper', '.solr_wrapper'
```

Comment:
Renamed the file in blacklight root and updated the generator method. Now copies it to the bl based app as .solr_wrapper.yml